app/helpers/feedback_messages_helper.rb

```diff
@@ -15,6 +15,17 @@ def offender_email_details
     { subject: "#{SiteConfig.community_name} Code of Conduct Violation", body: body }.freeze
   end
 
+  def reporter_email_details
+    body = <<~HEREDOC
```

Comment:
Suggestion: in some other places we assign the template string to a constant with placeholders and then use `format` to fill in the values, e.g.

```ruby
REPORTER_EMAIL_BODY = <<~HEREDOC
  Hi there,
  Thank you for flagging content that may be in violation of the %<community>s Code of Conduct and/or our Terms of Use. We are looking into your report and will take appropriate action.
  We appreciate your help as we work to foster a positive and inclusive environment for all!
  %<community>s Team
HEREDOC

def reporter_email_details
  body = format(REPORTER_EMAIL_BODY, community: SiteConfig.community_name)
  subject: "#{SiteConfig.community_name} Report", body: body }.freeze
end
```

Reply:
Interesting. Updated per your suggestion! Additionally, I addressed some inconsistencies with possessive/non-possessive proper nouns within all three of the email bodies. Let me know what you think. 😊